TerrariaServerAPI/TerrariaApi.Server/ServerApi.cs

```diff
@@ -78,7 +80,10 @@ static ServerApi()
 
 		internal static void Initialize(string[] commandLineArgs, Main game)
 		{
-
+			// https://github.com/Pryaxis/Plugins/security/advisories/GHSA-w3h6-j2gm-qf7q
```

Comment:
Are there plans to move this to it's own object at some point?  If not, I think there should be a dedicated function for InitializeListOfDangerousPlugins() instead of just doing that here.  I can imagine this being quite the mess as more and more plugins get added.

Reply:
I'm _really hoping_ we don't have to add more plugins to this list, but yeah you're right in terms of an initializer or separate object.